Sources/docc/DocCDocumentation.docc/other-formatting-options.md

```diff
@@ -0,0 +1,108 @@
+# Other Formatting Options
+
+Improve the presentation and structure of your content by incorporating special page elements.
+
+## Overview
+
+DocC offers support for various types of elements such as tables, notes, asides, and special characters.
+
+### Add Tables of Data
+
+To create a table, start a new paragraph and use dashes (`-`) to define 
+columns, and pipes (`|`) as column separators. Construction of a table requires 
+at least three rows: 
+
+* A row that contains the column names.
```

Comment:
Rewrite for clarity; for example:

The table's header row, which consists of pipes to separate the columns, and text for the headings of each column.

Or something to that effect.

Reply:
Rewritten in https://github.com/apple/swift-docc/pull/742/commits/0d0651d7a625986ec00e5b29ecdeae51053d2109